pycalphad/core/equilibrium.py

```diff
@@ -174,6 +175,11 @@ def equilibrium(dbf, comps, phases, conditions, output=None, model=None,
         Defaults to a pycalphad.core.solver.InteriorPointSolver.
     callables : dict, optional
         Pre-computed callable functions for equilibrium calculation.
+    global_min : bool, optional
+        Automatic miscibility gap detection ('global minimization').
+    user_starting_point : list of (phase name, dof) tuple
```

Comment:
Does having `user_start_point` as an argument to the high level `equilibrium` function make sense? It's probably meaningless to provide a starting point in this way for broadcasted conditions that are not close to each other. 

I've been chewing on this for awhile. My wish is to have an API written around calculating point equilibria that can efficiently be broadcasted to any set of conditions. Not sure how to accomplish this without a major rewrite, so I won't ask for that here.

Reply:
It might be possible, after refactoring `_solve_eq_at_conditions` per my previous comment, to only loop through conditions and call a "_solve_eq_at_condition" (no "s") function. 
Then you're just refactoring equilibrium to either take a broadcasted (`_solve_eq_at_conditions`) or unbroadcasted (`_solve_eq_at_condition`) code path.